configuration/editors.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: documentation
+title: Editors
+---
+
+{% include base.html %}
+
+# Editors - Different ways to power up your textual configuration
```

Comment:
Wdyt about "simplify" instead of "power up"?

Alternative:
"Different ways to make your textual configuration more comfortable"

(Ofc. Everything with correct headline capitalization then ;) )